src/pytorch_lightning/trainer/trainer.py

```diff
@@ -1965,10 +1965,15 @@ def predicting(self, val: bool) -> None:
 
     @property
     def tuning(self) -> bool:
+        rank_zero_deprecation("`trainer.tuning` property has been deprecated in v1.8.0 and will be removed in v1.10.0.")
```

Comment:
```suggestion
        rank_zero_deprecation("`Trainer.tuning` has been deprecated in v1.8.0 and will be removed in v1.10.0.")
```
For consistency with our other messages

Reply:
done!